Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1426,6 +1426,64 @@ void CFrame::ParseHotkeys()
 	{
 		State::Load(g_saveSlot);
 	}
+	if (IsHotkey(HK_TOGGLE_STEREO_SBS))
+	{
+		if (g_Config.iStereoMode != STEREO_SBS)
+		{
+			if (g_Config.sPostProcessingShader == "dubois")
+			{
+				g_Config.sPostProcessingShader = "";
+			}
+			g_Config.iStereoMode = STEREO_SBS;
+		}
+		else
+		{
+			g_Config.iStereoMode = STEREO_OFF;
+		}
+	}
+	if (IsHotkey(HK_TOGGLE_STEREO_TAB))
+	{
+		if (g_Config.iStereoMode != STEREO_TAB)
+		{
+			if (g_Config.sPostProcessingShader == "dubois")
+			{
+				g_Config.sPostProcessingShader = "";
+			}
+			g_Config.iStereoMode = STEREO_TAB;
+		}
+		else
+		{
+			g_Config.iStereoMode = STEREO_OFF;
+		}
+	}
+	if (IsHotkey(HK_TOGGLE_STEREO_ANAGLYPH))
+	{
+		if (g_Config.iStereoMode != STEREO_ANAGLYPH)
+		{
+			g_Config.iStereoMode = STEREO_ANAGLYPH;
+			g_Config.sPostProcessingShader = "dubois";
```

Comment:
This is the kind of question I was afraid of. I'm really a novice at C++, the fact I got this working in the first place was tha fact that VS 2013 has exellent search capabilities and I figured out that setting the anaglyph shader in Graphics/Settings in fact sets "dubois" in a similar manner as here.
This also seemed a rather straightforward way? But if you can think of another or a more abstract way to set the stereoscopy mode & required shader please tell ;)